Library/Plan.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Recurly
 {
-    public class Plan : RecurlyEntity
+    public class Plan : RevRecEntity
```

Comment:
Why is this changing? Is there still a `RecurlyEntity` being used?

Reply:
Yes, if you look at the thing taking its place, `RevRecEntity`, you will see that _it_ is inheriting from `RecurlyEntity`, so the new class is just fitted "in between" the original two.

Reply:
Ohhh okay. Cool. That makes sense.